app/experimenter/visualization/tests/api/test_views.py

```diff
@@ -125,14 +126,23 @@ def test_analysis_results_view_data(self, status, mock_exists, mock_open):
             "weekly": FORMATTED_DATA_WITHOUT_POPULATION_PERCENTAGE,
             "overall": FORMATTED_DATA_WITH_POPULATION_PERCENTAGE,
             "other_metrics": {"some_count": "Some Count"},
+            "metadata": {},
             "show_analysis": False,
         }
 
         class File:
+            def __init__(self, filename):
+                self.name = filename
+
             def read(self):
+                if "metadata" in self.name:
+                    return "{}"
```

Comment:
Hrmm yeah, in theory that'll work for tests just gotta make sure we don't accidentally set that off with a misnamed non metadata file.